### DISCUSSION TITLE
How to wrap complex react libs?

### DISCUSSION THREAD

**Discussion by lolitsicy** [reflex-dev/reflex#681] [Q&A]
Hi there, I absolutely love this framework but I'm extremely new to pynecone and literally have no knowledge on React or JS to be able to answer this question. I am trying to use the reactflow lib to create node like objects and I don’t know how to wrap the lib and use the methods as I am still struggling to understand the methods in the react lib.

All I want to so is be able to create and connect the nodes using pynecone. Any help would be great. 

Link to the React lib for reference: https://www.npmjs.com/package/reactflow

Any help would be great!

**Comment by Alek99** [comment]
Got it just responded on discord but will copy the message here for visibility 

----

We don't support the ref yet to add seekTo and getDuration but will add support for this you can implement the call backs though onReady see the color picker examples get_controlled_triggers code
